app/src/main/res/layout/mpower2_add_note.xml

```diff
@@ -0,0 +1,81 @@
+<?xml version="1.0" encoding="utf-8"?>
+<android.support.constraint.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent" android:layout_height="match_parent">
+
+    <TextView
+        android:id="@+id/add_note_label"
+        style="@style/Widget.MPower.TextView.SelectionSectionTitle"
+        android:layout_width="0dp"
+        android:layout_height="wrap_content"
+        android:layout_marginEnd="28dp"
+        android:layout_marginStart="28dp"
+        android:layout_marginTop="12dp"
+        android:gravity="center"
+        android:text="Add a note"
+        android:textStyle="bold"
+        app:layout_constraintEnd_toEndOf="parent"
+        app:layout_constraintStart_toStartOf="parent"
+        app:layout_constraintTop_toBottomOf="@+id/rs2_step_navigation_action_backward" />
+
+    <org.sagebionetworks.research.mobile_ui.widget.ActionButton
+        android:id="@+id/rs2_step_navigation_action_backward"
+        android:layout_width="32dp"
+        android:layout_height="0dp"
+        android:layout_marginStart="20dp"
+        android:layout_marginTop="20dp"
+        android:background="@drawable/mpower2_purple_arrow"
+        app:layout_constraintDimensionRatio="H,1:1"
```

Comment:
Never seen this before, do you have a link to documentation on this?  Seems pretty cool.  Historically we use LinearLayout and layout_wight to accomplish this

Reply:
https://developer.android.com/reference/android/support/constraint/ConstraintLayout It is about 1/3 to 1/2 of the way down under the Ratio subsection.

Reply:
That's so great